nixos/modules/services/networking/bird-lg.nix

```diff
@@ -0,0 +1,260 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.bird-lg;
+in
+{
+  options = {
+    services.bird-lg = {
```

Comment:
Missing an `enable` option.

Reply:
You can enable each subservice (frontend & proxy) separately, I don't think a general `enable` is required

Reply:
👍